### ISSUE TITLE
Charlie the Chicken NPC

### ISSUE THREAD

**Issue by bahbahrah** [StardewModders/mod-ideas#509]
I used Mr. Ginger mod and think it would be good if we also have Shane's chicken buddy mod. :D

Below are some of my ideas but it's up to the creator to follow it or not.

Story:  It was when Shane wanna make an omelet for his breakfast. Charlie suddenly popped up from its shell, and she thinks Shane is her parent. Since then she always following Shane wherever he goes. (Saloon, JojaMart, Festivals, or even in Shane's heart events...)

Personality: Charlie always follows Shane. She ignores everyone but Shane. She is a good girl in front of Shane but a bit rude to others especially when they talking behind Shane. Although you have 10 hearts with her, she still mostly ignores you and cares only Shane.

All her heart events are about Shane and Charlie's love development. :P

**Comment by lemurkat** [comment]
I don't think it would be that hard to make a chicken NPC. At least the dialogue would be easy.
(My husband wanted me to make a terrible goose.)
Having her move in if you were to marry Shane would be hard though.

**Comment by Caco-o-sapo** [comment]
For a moment I thought that you wanted my NPC to turn into a chicken, but the idea is super cool

**Comment by lemurkat** [comment]
Yes I would call her ChookCharlie in the files to avoid naming issues!

**Comment by lemurkat** [comment]
I am still really seriously contemplating this one. I wonder if I could make her live in the coop?
